cmake/Modules/FindARM.cmake

```diff
@@ -41,9 +41,15 @@ IF(CMAKE_SYSTEM_NAME MATCHES "Linux")
    ENDIF (OMAP4_TRUE)
 
 ELSEIF(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+   EXEC_PROGRAM("/usr/sbin/sysctl -n hw.optional.arm64" OUTPUT_VARIABLE
+      IS_ARM64)
+   IF(IS_ARM64 STREQUAL "1")
+      set(NEON_FOUND true CACHE BOOL "NEON available on ARM64")
```

Comment:
question: if we are setting NEON_FOUND here, we no longer need the checks below, yes?

Reply:
Correct, but we'll skip those by checking by `IF(NOT CPUINFO STREQUAL "")` check